### DISCUSSION TITLE
How to set textbox not keep focus when  quickly to next line?

### DISCUSSION THREAD

**Discussion by nanqic** [SubtitleEdit/subtitleedit#8032] [Q&A] [answered]
When using Subtitle Edit to quickly navigate to the next/previous line (by pressing ALT+ARROW UP/DOWN), 
is there a way to prevent the textbox from auto-focusing?
 I only need to adjust the timeline, not the text. Thank you!

**Comment by niksedk** [comment, marked as answer]
Try `Shift+Enter`, but you can set a custom shortcut if you want.
